demos/icon-toggle.html

```diff
@@ -24,11 +24,6 @@
         background: #303030;
       }
 
-      .mdc-theme--dark:not(.demo-color-combo) {
```

Comment:
Will do. Would adding left padding to all icons so that they are still aligned be an acceptable solution?